File: Euclid/Messages/Incoming/TRYFLAT.cs
using Euclid.Game;
using Euclid.Messages.Outgoing;
using Euclid.Network.Streams;

namespace Euclid.Messages.Incoming
{
    class TRYFLAT : IMessageEvent
    {
        public void Handle(Player player, Request request)
        {
            player.Send(new FLAT_LETIN());
        }
    }
}

File: euclid/src/main/java/euclid/messages/incoming/TRYFLAT.java
package euclid.messages.incoming;

import euclid.game.Player;
import euclid.messages.IMessageEvent;
import euclid.messages.outgoing.FLAT_LETIN;
import euclid.network.streams.Request;

public class TRYFLAT implements IMessageEvent {

    @Override
    public void handle(Player player, Request request) {
        player.send(new FLAT_LETIN());
    }
}
